README.md

```diff
@@ -2,17 +2,28 @@
 
 # loki-stack-app chart
 
-Giant Swarm offers a loki-stack Managed App which can be installed in tenant clusters.
-Here we define the loki chart with its templates and default configuration.
+Giant Swarm offers a loki-stack Managed App which can be installed in Kubernetes clusters.
```

Comment:
I would not call it "Managed" and why even mention Giant Swarm here? Just describe what it is: a chart that installs a minimal loki stack incl promtail and loki

Reply:
OK, this decription comes from "pre-agreement" catalogs layout. I think we should avoid "managed" and "giant swarm" in playground in general